pom.xml

```diff
@@ -293,6 +293,12 @@
                     <configuration>
                         <argLine>${argLine} -Duser.timezone=UTC -Dddf.version=${ddf.version}
                         </argLine>
+                        <includes>
+                            <include>**/*Test.java</include>
+                            <include>**/Test*.java</include>
+                            <include>**/*TestCase.java</include>
+                            <include>**/*Spec.class</include>
```

Comment:
I also see some groovy unit tests that currently don't match the convention
>Must have the same name as the class they are testing and be suffixed with Spec, e.g., MyClassSpec.groovy
https://codice.atlassian.net/wiki/display/DDF/Unit+Testing